e2etests/test/pnp_commands.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+const debug = require('debug')('e2etests:pnp_commands');
+const ServiceClient = require('azure-iothub').Client;
+const deviceMqtt = require('azure-iot-device-mqtt');
+const DeviceIdentityHelper = require('./device_identity_helper.js');
+const createDeviceClient = require('./testUtils.js').createDeviceClient;
+const assert = require('chai').assert;
+
+const hubConnectionString = process.env.IOTHUB_CONNECTION_STRING;
+const commandName = 'fakeCommand';
+const componentName = 'fakeComponent';
+const requestPayload = {this: 'is the payload for the request'};
+const responsePayload = {payload: 'for the response'};
+const statusCode = 200;
+
+[
+  DeviceIdentityHelper.createDeviceWithSas,
```

Comment:
Do we want to expand these tests to also run with module clients?

Reply:
Yes, I've refactored things so the tests I wrote will work for both `Client` and `ModuleClient`